tpcx_bb/queries/q01/tpcx_bb_query_01_sql.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
```

Comment:
```suggestion
```

Reply:
Removed. Then from now all the PRs will have only one space after ` import sys ` statement, as below patron.

```
import sys

from blazingsql import BlazingContext
```

Reply:
thanks!